projects/storefrontlib/src/cms-components/organization/units/unit-address-form/unit-address-form.component.spec.ts

```diff
@@ -106,6 +106,17 @@ class MockUrlPipe implements PipeTransform {
   transform() {}
 }
 
+@Component({
```

Comment:
There is no need to mock this component. You can import it normally and add it to the `imports` array.